### DISCUSSION TITLE
how to solve question-answer solution to do?

### DISCUSSION THREAD

**Discussion by heroesf** [rubenlagus/TelegramBots#981] [Q&A] [answered]
Or picture response.
Send picture.
Checks for an image.
Verifies that it was sent successfully.
If all is well, send a message thanking you for sending the picture.

I'm not a programmer, I'm just learning from a hobby.

**Comment by yiesko** [comment]
a

**Reply by heroesf** [reply]
?

**Comment by heroesf** [comment, marked as answer]
Problem solved.
